session/mysql: ErrorfContext uses %w and fails go vet

## Describe the bug

`session/mysql` 的两个异步持久化 worker 使用了：

```go
log.ErrorfContext(ctx, "async persist event failed: %w", err)
```

`%w` 是 `fmt.Errorf` 的错误包装指令，不受 `ErrorfContext` 日志 API 支持。因此 Go printf analyzer 会报错，导致该子模块的默认测试和 vet 失败。

`main@bcd0a567` 中仍存在相同代码：

- https://github.com/trpc-group/trpc-agent-go/blob/bcd0a5679624b85810dd6e047eb63ec8867a05f4/session/mysql/service.go#L865
- https://github.com/trpc-group/trpc-agent-go/blob/bcd0a5679624b85810dd6e047eb63ec8867a05f4/session/mysql/service.go#L897

## To reproduce

在仓库中执行：

```bash
cd session/mysql
go test ./...
```

或者在依赖该发布模块的消费者模块中执行：

```bash
go test trpc.group/trpc-go/trpc-agent-go/session/mysql
```

实际输出：

```text
service.go:865:36: ErrorfContext does not support error-wrapping directive %w
service.go:897:36: ErrorfContext does not support error-wrapping directive %w
```

执行 `go test -vet=off ./...` 可以通过，说明失败来自 vet 的日志格式检查。

## Expected behavior

`session/mysql` 应默认通过 `go test ./...` 和 `go vet ./...`。日志调用应使用 `%v`、`%s` 或结构化错误字段，而不是 `%w`。

## Environment

- Module: `trpc.group/trpc-go/trpc-agent-go/session/mysql v1.11.2`
- Also present in: `main@bcd0a567`
- OS: Windows/amd64
- Verified with Go: 1.21.13 and 1.26.4
- MySQL server is not required to reproduce this vet failure

--- a/session/mysql/service.go
+++ b/session/mysql/service.go
@@ -862,7 +862,7 @@ func (s *Service) startAsyncPersistWorker() {
 				if err := s.addEvent(ctx, eventPair.key, eventPair.event); err != nil {
 					log.ErrorfContext(
 						ctx,
-						"async persist event failed: %w",
+						"async persist event failed: %v",
 						err,
 					)
 				}
@@ -894,7 +894,7 @@ func (s *Service) startAsyncPersistWorker() {
 				if err := s.addTrackEvent(ctx, trackEventPair.key, trackEventPair.event); err != nil {
 					log.ErrorfContext(
 						ctx,
-						"async persist event failed: %w",
+						"async persist event failed: %v",
 						err,
 					)
 				}

